packages/lodestar/src/network/gossip/validator.ts

```diff
@@ -175,8 +180,8 @@ export class GossipMessageValidator implements IGossipMessageValidator {
       return false;
     }
 
-    const attestorIndices = getAttestingIndices(this.config, state, attestationData, aggregate.aggregationBits);
-    if (!attestorIndices.includes(aggregatorIndex)) {
+    const committee = getBeaconCommittee(this.config, state, attestationData.slot, attestationData.index);
```

Comment:
We should have access to the committee in our `chain.epochCtx` cache.

I bet that we have many places in the gossip validation logic that could be refactored to use chain.epochCtx, might be worth tackling as as separate PR/issue.

Reply:
I only see `getBeaconProposer` and `getBeaconCommittee` in EpochContext that can be applied so I applied that to this PR